modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/product_card/product_image.jsp

```diff
@@ -20,9 +20,6 @@
 
 <c:if test="<%= cpContentListEntryModel.getSkuId() != 0 %>">
 	<div class="aspect-ratio-item-bottom-left">
-		<commerce-ui:availability-label
-			lowStock="<%= cpContentListEntryModel.isLowStock() %>"
-			stockQuantity="<%= cpContentListEntryModel.getStockQuantity() %>"
-		/>
+		<commerce-ui:availability-label />
```

Comment:
@ivicac @gianmarcobrunialti  instead of getting the catalogentry from the request we should pass it as param